reactor-test/src/main/java/reactor/test/StepVerifier.java

```diff
@@ -450,6 +450,21 @@ static void resetDefaultTimeout() {
 		 */
 		StepVerifier expectErrorSatisfies(Consumer<Throwable> assertionConsumer);
 
+		/**
+		 * Verify that the {@link Publisher} under test doesn't terminate but
+		 * rather times out after the provided {@link Duration} (a timeout implying
+		 * a cancellation of the source).
+		 * <p>
+		 * This is equivalent to appending the {@link reactor.core.publisher.Flux#timeout(Duration) timeout}
+		 * operator to the publisher and expecting a {@link java.util.concurrent.TimeoutException}
+		 * {@link #expectError(Class) onError signal}, while also triggering a {@link Step#thenAwait(Duration) wait}
+		 * to ensure unexpected signals are detected.
+		 *
+		 * @param duration the {@link Duration} for which no new event is expected
+		 * @return the built verification scenario, ready to be verified
+		 */
+		StepVerifier expectTimeout(Duration duration);
```

Comment:
is it chainable? (do we expect the users to call more `expect*` after `expectTimeout`?)
Or maybe we only need the `verifyTimeout` one?

Reply:
no it is not chainable, this is intended with the same usage pattern as `expectError`.